packages/feedback/src/integration.ts

```diff
@@ -1,4 +1,5 @@
-import type { Integration, IntegrationFn } from '@sentry/types';
+import { type replayIntegration } from '@sentry/replay';
```

Comment:
let's not use this syntax, but `import type {}` - I've read that this is better/easier to treeshake!

Also, generally, I'd say maybe we should avoid this - we have this as devDependency only for the type here, which is not really correct from a bundling perspective 😬 I'd rather we just inline the relevant type here (or if really needed put some interface type in `@sentry/types` that both packages can use)?

Reply:
yeah, that's always the problem with the types stuff and the circular dependency stuff 😬 

So I'd go with one of these approaches - no strong feelings:

1. We add a minimal interface to types - IMHO that may be OK to do for the integration itself, as the surface is quite small and can be considered the public contract. then in replay we can do `class Replay implements ReplayIntegrationInterface` or something like this 🤔 
2. We inline the styles, which is obv. not super secure but probably OK/fine here (as tests would fail anyhow hopefully if that would change), and we only use a single method right now?